contrib/docker-compose.yaml

```diff
@@ -28,9 +28,6 @@ services:
       -rest
       -rpcbind=0.0.0.0:18443
       -rpcallowip=0.0.0.0/0
-    post_start:
```

Comment:
this was moved to `load_bitcoin_wallet_script` because `post_start` hook requires docker compose 2.30.0 or never and the `ubuntu-latest` uses 2.27.0